### ISSUE TITLE
Get-ESXPatching - could not find any baseline(s)

### ISSUE THREAD

**Issue by joffer77** [arielsanchezmora/vDocumentation#36]
Normally I don't have the predefined VUM baselines attached, I create my own. I tried with and without those predefined baselines attached to my cluster (or datacenter), but I get this error in all cases (vDocumentation v2.4.0):

`
PS C:\> Get-ESXPatching -ExportExcel -folderpath $vDocExportPath
Get-ESXPatching : Could not find any baseline(s) named Critical Host Patches*,Non-Critical Host Patches* on server vcenter.domain.local. Please check Baseline name and try again.
At line:1 char:1
+ Get-ESXPatching -ExportExcel -folderpath $vDocExportPath
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Get-ESXPatching`

Running PowerShell 5.1 on WinSrv 2016 1607

**Comment by edmsanchez** [comment]
Hi joffer77; please test on latest version. Get-ESXPatching has a -baseline parameter that you can use to specify the name(s) of your custom baseline(s). 
Example:
Get-ESXPatching -cluster production-cluster -baseline 'Custom baseline Host Patches'

**Comment by joffer77** [comment]
I tried now with v2.4.2 and without -baseline parameter. This time it worked, and by default it used the predefined baselines. I also tried with my own custom baselines and hat was also fine.

**Comment by edmsanchez** [comment]
thanks for the feedback, let me try to reproduce on my end. The custom baseline you are using, includes only VMware patches right?

**Comment by joffer77** [comment]
Yes, only VMware patches. I've just created a custom baseline that only includes 6.5 patches instead of all patches for versions we don't use.

**Comment by arielsanchezmora** [comment]
@edmsanchez do you think it's spaces or some other dumb thing that isn't being passed on correctly?
